portal/templates/staff_profile_create.html

```diff
@@ -62,6 +62,7 @@ <h4 class="tnth-headline left-indent-top">{{ _("New Staff") }}</h4>
 {% endblock %}
 {% block document_ready %}
     var orgList = {% if org_list %} {{org_list | safe}} {% else %} false {% endif %};
+    var OT = new OrgTool(); OT.init();
```

Comment:
If you had an element you could check for the presence of (eg <`div id="orgList">`), could you move this initialization into `main.js`?

Reply:
@ivan-c Ivan, yeah, I see this and  some inline JS still lingering in some template files.  My next step is to clean those up as well.  Expecting a PR soon on this.